src/vcSettings.h

```diff
@@ -366,14 +367,19 @@ struct vcSettings
     vcTileRendererMapQuality mapQuality;
     vcTileRendererFlags mapOptions;
 
-    float mapHeight;
+    struct Layer
+    {
+      bool enabled;
+      float mapHeight;
 
-    char mapType[32]; // 'custom', 'euc-osm-base', 'euc-az-aerial', 'euc-az-roads'
-    vcMapServer customServer;
-    vcMapServer activeServer; // The server settings actually in use
+      char mapType[32]; // 'custom', 'euc-osm-base', 'euc-az-aerial', 'euc-az-roads'
+      vcMapServer customServer;
+      vcMapServer activeServer; // The server settings actually in use
 
-    vcMapTileBlendMode blendMode;
-    float transparency;
+      vcMapTileBlendMode blendMode;
+      float transparency;
+    } layers[vcMaxTileLayerCount];
+    int activeLayerCount;
```

Comment:
What does this provide over checking if the layer is `enabled`?

Reply:
`activeLayerCount`? Just simplifies some logic in parts.